ofrak_core/ofrak/core/elf/model.py

```diff
@@ -240,6 +242,9 @@ class ElfProgramHeaderType(Enum):
     SHLIB = 5
     PHDR = 6
     TLS = 7
+    GNU_EH_FRAME = 0x6474E550
+    GNU_STACK = 0x6474E551
+    GNU_RELRO = 0x6474E552
```

Comment:
Where do these numbers come from? A comment with a citation would be useful.

Reply:
Perhaps something like this?

<https://github.com/torvalds/linux/blob/ceaa837f96adb69c0df0397937cd74991d5d821a/include/uapi/linux/elf.h#L25-L41>

Reply:
```suggestion
    # Magic numbers from:
    # https://elixir.bootlin.com/linux/latest/source/include/uapi/linux/elf.h#L38
    GNU_EH_FRAME = 0x6474E550
    GNU_STACK = 0x6474E551
    GNU_RELRO = 0x6474E552
```

Reply:
Added a link in the docstring